eZ/Publish/Core/settings/storage_engines/cache.yml

```diff
@@ -18,16 +18,23 @@ parameters:
     ezpublish.spi.persistence.cache.persistenceLogger.class: eZ\Publish\Core\Persistence\Cache\PersistenceLogger
     # Make sure logging is only enabled for debug by default
     ezpublish.spi.persistence.cache.persistenceLogger.enableCallLogging: "%kernel.debug%"
-    # Global in-memory settings
+    # Global in-memory settings, for meta data
     ezpublish.spi.persistence.cache.inmemory.ttl: 3000
     ezpublish.spi.persistence.cache.inmemory.limit: 100
     ezpublish.spi.persistence.cache.inmemory.enable: true
+    # Global in-memory settings, for content in-memory cache
+    ## TTL on purpose low to avoid getting outdated data in prod, in dev it will be x2 to account for debug overhead
+    ezpublish.spi.persistence.cache.inmemory.content.ttl: 400
```

Comment:
I'm wondering if this value might somehow depend on the content itself in the future e.g. content from "Media" is rarely changed in comparison to "Content" section so might live longer in the cache.

Reply:
:thinking: you're somewhat right, but probably won't be applicable to everyone _(as in might be wrong assumption)_, and might not be worth it to distinguish it.